module/behaviour/planning/KickPlanner/src/KickPlanner.cpp

```diff
@@ -92,12 +99,13 @@ namespace behaviour {
                 }
             });
 
-            on<Trigger<Ball>, With<Field>, With<FieldDescription>, With<KickPlan>, With<Sensors>>().then(
-                [this](const Ball& ball,
-                       const Field& field,
-                       const FieldDescription& fd,
-                       const KickPlan& kickPlan,
-                       const Sensors& sensors) {
+            on<Trigger<Ball>, With<Field>, With<FieldDescription>, With<KickPlan>, With<GameState>, With<Sensors>>()
```

Comment:
Make the game state optional for this, we still want to be able to kick balls when testing without a game controller.